### DISCUSSION TITLE
How to delete or reset organization?

### DISCUSSION THREAD

**Discussion by luizen** [Kong/insomnia#7101] [Help]
I have logged in with SSO using company email and created a personal organization (don't remember if this was created automatically or I did it). I created some projects and was messing up with them to learn how Insomnia works, its features, etc. I realized there are so many bugs when trying to sync, delete, move, rename projects and collections that I decided to uninstall everything and start from scratch. However, I'd like to delete all projects inside my workspace/organization, but it seems this is not possible. 
How to achieve that?

The collections inside my projects are gone as well, they seem to be on the server, but nothing gets downloaded to the client. So that's one more reason I'd like to delete everything.

**Comment by notjaywu** [comment]
Hi luizen, could you join this Slack channel (https://join.slack.com/t/getinsomnia/shared_invite/zt-2dkfcuyk5-CPX9ij7Pjx1B68pcf4l_qg) so I can DM you for more details.

**Reply by benedicttobias** [reply]
> Hi luizen, could you join this Slack channel (https://join.slack.com/t/getinsomnia/shared_invite/zt-2dkfcuyk5-CPX9ij7Pjx1B68pcf4l_qg) so I can DM you for more details.

Hi @notjaywu , is there any reason the step is not public and needs to be in the Slack? I do have the same situation.

**Reply by notjaywu** [reply]
No, it’s simply to ensure we can exchange confidential information safely. We want to avoid having you share any private account details publicly.

What is the problem?
